crates/ruff_linter/src/rules/flake8_boolean_trap/rules/boolean_type_hint_positional_argument.rs

```diff
@@ -96,6 +97,40 @@ impl Violation for BooleanTypeHintPositionalArgument {
     }
 }
 
+fn match_annotation_to_bool(annotation: &Expr, semantic: &SemanticModel) -> bool {
+    // check for both bool (python class) and 'bool' (string annotation)
+    let hint = match annotation {
+        Expr::Name(name) => &name.id == "bool",
+        Expr::Constant(ast::ExprConstant {
+            value: Constant::Str(ast::StringConstant { value, .. }),
+            ..
+        }) => value == "bool",
```

Comment:
With https://github.com/astral-sh/ruff/pull/8064 merged, you'll need to update this to use `Expr::StringLiteral` instead:

```suggestion
        Expr::StringLiteral(ast::ExprStringLiteral { value, .. }) => value == "bool",
```